index.html

```diff
@@ -2898,7 +2918,7 @@ <h3>The <dfn>PropertyReadHandler</dfn> callback</h3>
           </li>
           <li>
             Save the request sender information together with |options| to
-            |property|'s <a>internal observer list</a>, in order to be able
+            {{ExposedThing/[[propertyObservers]]}}'s |name| value, in order to be able
```

Comment:
If the above is changed (use a map of sets for the internal slot), the original text is easier to get and more correct.

> [=Append|list-append=] the request sender information together with |options| to |property|'s <a>internal observer list</a>